pkg/cadvisor/waitfordocker.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/ash
```

Comment:
please use `/bin/sh` not explicitly choosing `ash`

Reply:
oh sure, I tried too hard to silence shellcheck.